src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -1,28 +1,33 @@
 namespace FSharp.Control
 
+open System.Collections.Generic
+open System.Threading.Tasks
+
 #nowarn "1204"
 
-module TaskSeq =
-    open System.Collections.Generic
-    open System.Threading.Tasks
+[<AutoOpen>]
+module TaskSeqExtensions =
+    module TaskSeq =
+        /// Initialize an empty task sequence.
+        val empty<'T> : taskSeq<'T>
 
-    /// Initialize an empty taskSeq.
-    val empty<'T> : taskSeq<'T>
+[<Sealed>]
```

Comment:
I beieve it needs AbstractClass and private ctor too? https://stackoverflow.com/a/13419343/11635

Reply:
Funny reading my own comment under the 2nd answer there, which in turn leads to this bug report: https://github.com/dotnet/fsharp/issues/8093

Reply:
Done :)